src/ci/bin/test.inc.sh

```diff
@@ -103,8 +116,13 @@ cromwell::private::create_build_variables() {
     git_commit_message="$(git log --format=%B --max-count=1 HEAD 2>/dev/null || true)"
     if [[ "${git_commit_message}" == *"[force ci]"* ]]; then
         CROMWELL_BUILD_FORCE_TESTS=true
-    else
+        CROMWELL_BUILD_MINIMAL_TESTS=false
+    elif [[ "${git_commit_message}" == *"[minimal ci]"* ]]; then
```

Comment:
`[a dusting of ci]`

Reply:
One second... I'm going to move this to another PR so it doesn't get the previous thumbs grandfathered in.

Reply:
Moved to https://github.com/broadinstitute/cromwell/pull/5488